include/eve/module/core/regular/impl/minmax.hpp

```diff
@@ -0,0 +1,91 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/platform.hpp>
+#include <eve/concept/value.hpp>
+#include <eve/detail/implementation.hpp>
+#include <eve/module/core/regular/max.hpp>
+#include <eve/module/core/regular/min.hpp>
+#include <eve/traits.hpp>
+
+namespace eve::detail
+{
+template<typename T>
+constexpr bool has_native_minmax() noexcept
+{
+  using e_t = element_type_t<T>;
+  constexpr bool small_int = !(std::integral<e_t> && sizeof(e_t) == 8);
```

Comment:
This is hard to follow but failry obviously wrong since it never takes the number of elements into account